index.css

```diff
@@ -1,17 +1,173 @@
 /* COMEÇE POR AQUI */
 /* Celulares (para telas entre 360px e 576px) */
 @media (min-width: 360px) and (max-width: 576px) {
-   
+   #head{
+    margin-top: 50px;
+    font-size: 1.5rem;
+    font-weight: 800;
+   }
+   #head-grad{
```

Comment:
Utilize um padrão de nome de classes, head-grad ou headGrad estão corretos, mas utilize eles para todas classes que for declarar